core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ConfigResource.java

```diff
@@ -235,6 +235,9 @@ public ConfigUpdateRequest updateTopLevelConfig(
         throw new RestInvalidRuleSetException(e.getMessage());
       }
     }
+    if (request.isValidateFields() == null) {
+      request.setValidateFields(schemaRegistry.isDefaultValidateFields());
+    }
```

Comment:
Remove these lines.  The default should be used in `isSchemaFieldValidationEnabled` instead.